sc/src/main/java/uk/ac/ebi/atlas/search/AutocompleteController.java

```diff
@@ -51,9 +53,18 @@ public AutocompleteController(SolrBioentitiesSuggesterService suggesterService,
     public String fetchTopSuggestions(
             @RequestParam(value = "query") String query,
             @RequestParam(value = "species", required = false, defaultValue = "") String species,
-            @RequestParam(value = "suggestCount", required = false, defaultValue = "15") int suggestCount) {
+            @RequestParam(value = "suggestCount", required = false, defaultValue = "20") int suggestCount) {
+
+        String[] splitSpecies = species.split(",");
         return GSON.toJson(
-                suggesterService.fetchBioentitySuggestions(query, speciesFactory.create(species), suggestCount));
+                Arrays.stream(splitSpecies)
```

Comment:
Was it possible to pass multiple species to the `suggesterService` and let Solr handle the processing, rather than doing it in the controller?

Reply:
I thought you could, then I tried (by using commas) and realised that it ignores it, and now it just occurred to me to try with spaces and... lo and behold! This works:
```
http://ves-hx-69:8983/solr/bioentities/suggest?wt=json&indent=on&suggest.dictionary=bioentitySuggester&suggest.q=aspm&suggest.cfq=Mus_musculus%20Homo_sapiens%20Macaca_mulatta
```

Reply:
Will change it...

Reply:
I wouldn't say it's a massive priority! It will involve quite a lot of fiddling I imagine.